dashboard/app/main.go

```diff
@@ -64,6 +64,7 @@ func initHTTPHandlers() {
 		http.Handle("/"+ns+"/repos", handlerWrapper(handleRepos))
 		http.Handle("/"+ns+"/bug-stats", handlerWrapper(handleBugStats))
 		http.Handle("/"+ns+"/subsystems", handlerWrapper(handleSubsystemsList))
+		http.Handle("/"+ns+"/backports", handlerWrapper(handleBackports))
```

Comment:
Let's selectively enable it.

Reply:
I've updated the PR so that we don't display the link in the header if there are no backports. It's fine to keep the handler here, it will output nothing in this case.